classroom.rb

```diff
@@ -0,0 +1,12 @@
+class Classroom
+  attr_accessor :label
+
+  def initialize(label)
+    @label = label
+    @students = []
+  end
+
+  def add_student(student)
+    @students << student
+  end
```

Comment:
- [ ] Please add getters `attr_reader` for students here in Classroom class to make sure the has-many / belongs-to relationships between Classroom and Student are created properly.

Reply:
Oh sure. My bad. 🥲